docs/DeveloperGuide.md

```diff
@@ -749,49 +767,227 @@ _Customization of Contacts_
 
 Given below are instructions to test the app manually.
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** These instructions only provide a starting point for testers to work on;
-testers are expected to do more *exploratory* testing.
-
+<div markdown="span" class="alert alert-info">:information_source: **Note:** The instructions and test cases under this section only provide the general aspects of the SociaLite app for testers to work on. If you happen to be a tester, please feel free to explore around using your own test cases!
 </div>
+### Launch and exiting the app
+
+Download the latest `jar` file from SociaLite [release page](https://github.com/AY2122S1-CS2103T-F11-4/tp/releases), and copy it into an empty folder that you have access to read and write.
+
+1. Launching
+   1. Double-click on the `jar` file icon in your OS' GUI
+   
+   	Expected: the GUI for SociaLite app shows up with a set of sample contacts.
```

Comment:
Oops actually not entirely sure about how to do so! (cos it rendered fine on intellij...🥲)

<img width="509" alt="截圖 2021-11-07 下午12 07 44" src="https://user-images.githubusercontent.com/66581587/140631972-1dbeb0b4-598c-4c98-b555-d8337d624cfc.png">

Reply:
hmm i get the same as stanley when i view on github though. Perhaps it would be better to optimize for github because our testers would be viewing this on github?

Reply:
Haha maybe it's just a glitch on GItHub which happens to my docs sometimes too! As long as it's rendered correctly on GitHub pages, it should be fine!